rero_ils/modules/documents/dojson/contrib/jsontomarc21/model.py

```diff
@@ -361,7 +476,7 @@ def add_value(result, sub_tag, value):
 def add_values(result, sub_tag, values):
     """Add values with tag to result."""
     if values:
-        for count in range(len(values)):
+        for _ in range(len(values)):
```

Comment:
why using `range` ? seems completely useless

Reply:
If I have `values = [1, 2, 3]` and `sub_tag = 'x'` the marc21 representation in python dictionary should be:
```python
{
    '__order__': `'x', 'x', 'x',
    'x':  [1, 2, 3]
}